docs/user-guide/rules/highest-available-document-mode.md

```diff
@@ -283,6 +283,77 @@ Note that:
 
 </details>
 
+<details>
+<summary>How to configure IIS</summary>
+
+### Adding the `X-UA-Compatible` header
+
+To add the `X-UA-Compatible` header you can use a [`URL rewrite` rule][url rewrite]
+that matches the `text/html` content-type of a response and then adds it:
+
+```xml
+<configuration>
+     <system.webServer>
+        <rewrite>
+            <outboundRules>
+                <rule name="X-UA-Compatible header">
+                    <match serverVariable="RESPONSE_X_UA_Compatible" pattern=".*" />
+                    <conditions>
+                        <add input="{RESPONSE_CONTENT_TYPE}" pattern="^text/html" />
+                    </conditions>
+                    <action type="Rewrite" value="ie=edge"/>
+                </rule>
+            </outboundRules>
+        </rewrite>
+    </system.webServer>
+</configuration>
+```
+
+Note that the following snippet will add the header to all the responses
+regardless of the content type:
+
+```xml
+<configuration>
+     <system.webServer>
+        <httpProtocol>
+             <customHeaders>
+                <add name="X-UA-Compatible" value="IE=edge"/>
+             </customHeaders>
+         </httpProtocol>
+    </system.webServer>
+</configuration>
```

Comment:
I don't think we should include this example.

Reply:
OK